Be more explicit about cookie deletion (#8476)
withastro/docs: src/content/docs/en/reference/api-reference.mdx

**Type:** `(key: string, options?: CookieDeleteOptions) => void`
</p>

> Marks the cookie as deleted by setting the expiration date to 0 in Unix time. Once a cookie is deleted `Astro.cookies.has()` will return `false` and `Astro.cookies.get()` will return an [`AstroCookie`](#astrocookie) with a `value` of `undefined`. Options available when deleting a cookie are: `domain`, `path`, `httpOnly`, `sameSite`, and `secure`.

Reviewer: ```suggestion
Invalidates a cookie by setting the expiration date in the past (0 in Unix time).

Once a cookie is "deleted" (expired), `Astro.cookies.has()` will return `false` and `Astro.cookies.get()` will return an [`AstroCookie`](#astrocookie) with a `value` of `undefined`. Options available when deleting a cookie are: `domain`, `path`, `httpOnly`, `sameSite`, and `secure`.
```
Great call out, @whitep4nth3r ! In fact, I like your plain language explanation in your PR description and blog post, so I think it's great to include it here so people have another way to parse what "0 in Unix time" means.

What do you think about my suggestion above?

Author: This is great, your suggestion gives more context than mine! 👍🏼
